src/coreclr/tools/aot/crossgen2/Program.cs

```diff
@@ -330,6 +339,13 @@ private int Run(string[] args)
                 return 1;
             }
 
+            if (_commandLineOptions.Version)
+            {
+                string version = GetCompilerVersion();
+                Console.WriteLine("Crossgen2 Compiler Version is {0}.", version);
```

Comment:
```suggestion
                Console.WriteLine(version);
```
The other dotnet tools just print the version, without anything extra. For example:
```
jkotas@jkotas7:~$ dotnet --version
6.0.201
jkotas@jkotas7:~$ dotnet sos --version
6.0.257301+27172ce4d05e8a3b0ffdefd65f073d40a1b1fe54
```

Reply:
Will update the string. Thanks for the observation Jan.